src/main/resources/validationrules/stcic/stcic-judicial-office-holder-mapping.drl

```diff
@@ -47,7 +47,6 @@ then
       .endTime($jap.getEndTime())
       .regionId($jap.getRegionId())
       .baseLocationId($jap.getBaseLocationId())
-      .primaryLocation($jap.getPrimaryLocationId())
```

Comment:
The `JudicialAccessProfile` > `JudicialOfficeHolder` is an interim step.  So can we leave all the primaryLocation mappings in this file: but keep the removals in the `JudicialOfficeHolder` >`RoleAssignment` mappings.

i.e. I would like to keep all the JOH mappings the same across the services to ease the work for JRD eLinks changes.

Reply:
The JOH primary location mappings have been restored